Statistics/Distributions/LogNormal.cs

```diff
@@ -9,121 +9,111 @@ namespace Statistics.Distributions
 {
     public class LogNormal : IDistribution, Utilities.IValidate<LogNormal>
     {      
+        #region Fields and Properties
+        private double _mean;
+        private double _standardDeviation;
+        private double _min;
+        private double _max;
         internal IRange<double> _ProbabilityRange;
-        private MathNet.Numerics.Distributions.LogNormal _Distribution;
-        private bool _Constructed;
 
-        #region Properties
-        public IDistributionEnum Type => IDistributionEnum.LogNormal;
+        #region IDistribution Properties
+        public IDistributionEnum Type => IDistributionEnum.Normal;
         [Stored(Name = "Mean", type = typeof(double))]
-        public double Mean { get; set; }
-
-        public double Median => _Distribution.Median;
-
-        public double Mode => _Distribution.Mode;
-
-        public double Variance => _Distribution.Variance;
-        [Stored(Name = "St_Dev", type = typeof(double))]
-        public double StandardDeviation{ get; set; }
-
-        public double Skewness => _Distribution.Skewness;
-
-        public IRange<double> Range { get; set; }
-        [Stored(Name = "Min", type = typeof(double))]
-        public double Min
-        {
-            get; set;
-        }
+         public double Mean { get{return _mean;} set{_mean = value;} }
+        [Stored(Name = "Standard_Deviation", type = typeof(double))]
+        public double StandardDeviation { get{return _standardDeviation;} set{_standardDeviation = value;} }
+        [Stored(Name = "Min", type =typeof(double))]
+        public double Min { get{return _min;} set{_min = value;} }
         [Stored(Name = "Max", type = typeof(double))]
-        public double Max
-        {
-            get; set;
-        }
+        public double Max { get{return _max;} set{_max = value;} }
```

Comment:
what is the benefit of writing this out versus public double Max { get; } { set;} ?

Reply:
i can then specify the class private variable for max explicitly and i can control any behavior in get and set i want (like adding validation or notification of change)